catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -140,7 +139,7 @@
 
     @Before
     public void setUp() throws Exception {
-        framework = createFramework();
+        framework = createFramework(2);
```

Comment:
For a majority of tests, this case is used, so I only called createFramework when this needed an alternate value.  I can add a comment for clarity in those cases and overload the method so this remains 'createFramework()'.